flang/cmake/modules/FlangCommon.cmake

```diff
@@ -24,6 +24,13 @@ if (FLANG_RUNTIME_F128_MATH_LIB)
   add_compile_definitions(FLANG_RUNTIME_F128_MATH_LIB="${FLANG_RUNTIME_F128_MATH_LIB}")
 endif()
 
+# The NVPTX target can't emit a binary due to the PTXAS dependency, just
+# hard-code this.
+if ("${LLVM_RUNTIMES_TARGET}" MATCHES "^nvptx")
+  add_compile_definitions(FLANG_LITTLE_ENDIAN=1)
+  return()
```

Comment:
This being the last check in this file is just happenstance. Can you make it a regular if/else?